omero/users/cli/import-bulk.rst

```diff
@@ -0,0 +1,99 @@
+Using bulk imports
+==================
+
+The CLI import option ``--bulk`` specifies a configuration file that
+can be used to perform a batch of imports with the same or similar
+options. The file is written in a simple YAML syntax and can be placed
+anywhere.
+
+A simple YAML file might look like: ::
+
+    ---
+    continue: "true"
+    transfer: "ln_s"
+    path: "my-files.txt"
+
+Assuming that my-files.txt is a list of files such as ::
+
+    fileA
+    fileB
+    fileC
+
+this is equivalent to: ::
+
+    $ bin/omero import -k --transfer=ln_s fileA fileB fileC
+
+Bulk-only options
+-----------------
+
+Path
+^^^^
+
+The ``path`` key specifies a file from which *each individual line*
+will be processed as a separate import. In the simplest case, a single
+file is placed per line as above. For more complex usages, ``path``
+can point to a tab-separated value (TSV) file where each field will
+be interpreted based on ``columns``.
+
+Columns
+^^^^^^^
+
+A fairly regular requirement in importing many files is that for
```

Comment:
As all the examples below are relying on the common import options, should we move this section above e.g.

1- Common options (vanilla import)
2- Bulk specific options
 a- Path
 b- Columns (with examples including some of the common options)
 c- dry-run

Reply:
With "common options" I was trying to get across, "and you can use all the rest". Perhaps a new name and links to other documentation would make that clearer.

Reply:
Is it clearer now?